tests/test_rp_exec.py

```diff
@@ -62,6 +62,9 @@ async def test_raptor_master(pilot_description, rp_venv):
     if rp_venv is None:
         pytest.skip("This test requires a user-provided static RP venv.")
 
+    if pilot_description.access_schema == "fork":
```

Comment:
I'll have to check why this logic isn't getting triggered when it should...

Reply:
Doh! Of course not. This is the access_schema, not the scheme from the URL. Silly me. Need a little more logic...